src/clj/game/cards/resources.clj

```diff
@@ -993,7 +993,9 @@
                                                                   (count from) from) card nil)
                      (do (clear-wait-prompt state :corp)
                          (effect-completed state side eid card)))))
-    :trash-effect {:effect (effect (mill :runner 3))}}
+    :trash-effect {:msg (msg "trash " (join ", " (map :title (take 3 (:deck runner))))
+                             " from their Stack as Rolodex was trashed ")
```

Comment:
Pretty sure this reads (from memory) as "Player uses Rolodex to trash card a, card b from their Stack as Rolodex was trashed ", which doesn't look right to me.